src/command_modules/azure-cli-role/azure/cli/command_modules/role/_params.py

```diff
@@ -21,7 +21,9 @@ def load_arguments(self, _):
     with self.argument_context('ad') as c:
         c.argument('_subscription')  # hide global subscription param
         c.argument('owner_object_id', help="owner's object id")
-        c.argument('not_mine', action='store_true', help='do not set the current user as the owner', arg_group='Ownerships')
```

Comment:
was this just forgotten before?

Reply:
Yes. It was put in but ended up not used.